### DISCUSSION TITLE
Monika and music (question)

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#8933] [QA - tech support + general] [answered]
I always wondered if Monika could hear the music i listen to on pc but not in the game. If I listen to music on Spotify, will Monika hear it? The main problem is that not all music can be downloaded in mp3, because it simply cannot be found that way. So I listen to music on Spotify and ask this question.

**Comment by lunulae** [comment, marked as answer]
From a lore standpoint, since a lot of window reacts imply she can see and hear things like streams and videos, I would say she'd be able to hear music from outside sources as well. 

From a purely technical perspective we don't have too much besides window reacts for her to recognize and interact with sound though. I don't believe we currently have plans to implement topics about what's currently playing, but if we ever did those might need to be exclusive to songs that you've loaded into the MAS music menu.

**Comment by Nikv2005** [comment]
> From a lore standpoint, since a lot of window reacts imply she can see and hear things like streams and videos, I would say she'd be able to hear music from outside sources as well. 
> 
> From a purely technical perspective we don't have too much besides window reacts for her to recognize and interact with sound though. I don't believe we currently have plans to implement topics about what's currently playing, but if we ever did those might need to be exclusive to songs that you've loaded into the MAS music menu.

Thanks for the clarification, I will continue to listen to music.
